### PR TITLE
Court: Optimize config getters

### LINKED ISSUE #185: Config: Optimize getters for the Court
Currently, we are exposing one single getter retrieving the whole config every time. We could optimize this having different getters for each particular consumer

### PR DESCRIPTION
**Pull request by facuspagnuolo**
Fixes #185 

This PR includes the following changes:
- Optimized config getter for creating disputes
- Optimized config getter for drafting
- Make sure config is fetched only once in the rest of the Court entry points (a step towards #107)

### CODE REVIEW

File: contracts/court/Court.sol [hunk 1]
```diff
@@ -657,9 +666,14 @@ contract Court is ControlledRecoverable, ICRVotingOwner {
             uint256 confirmAppealDeposit
         )
     {
+        _checkRoundExists(_disputeId, _roundId);
```
Comment (bingen) [inline contracts/court/Court.sol]: :heart: :raised_hands:  (why this change now though?)

Comment (bingen) [reply contracts/court/Court.sol]: Also, why only here?

Comment (facuspagnuolo) [reply contracts/court/Court.sol]: stack too deep my friend 😓
